src/components/DropdownButton.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { useState, useRef, useCallback } from "react";
+import { useOnClickOutside } from "usehooks-ts";
+
+interface DropdownButtonProps {
+  buttonText: string;
+  children: React.ReactNode;
+}
+
+export const DropdownButton = (props: DropdownButtonProps) => {
+  const [isActive, setActive] = useState(false);
+  const toggleActive = () => setActive((x) => !x);
+  const exitButtonWrap = useRef(null);
+
+  useOnClickOutside(exitButtonWrap, () => setActive(false));
+
+  return (
+    <div className="dp-button-editor-wrap" ref={exitButtonWrap}>
+      <div className="dp-button-box">
+        <button
+          type="button"
+          className={
+            "dp-button button-medium dp-button-exit" +
+            (isActive ? " active" : "")
+          }
+          aria-controls="dp-exit-editor"
+          onClick={toggleActive}
```

Comment:
```suggestion
          onClick={setActive(!isActive)}
```
This is verbose and flat for me.

Reply:
♥️

Reply:
Fixed! Thanks @Cromeror and @andresmoschini